api/src/websocket/handlers/items.ts

```diff
@@ -0,0 +1,98 @@
+// import logger from '../../logger';
+import { getSchema } from '../../utils/get-schema';
+import { ItemsService, MetaService } from '../../services';
+import type { WebSocketClient } from '../types';
+import { fmtMessage } from '../utils/message';
+import emitter from '../../emitter';
+import { sanitizeQuery } from '../../utils/sanitize-query';
+import { handleWebsocketException, WebSocketException } from '../exceptions';
+import { WebSocketItemsMessage } from '../messages';
+
+export class ItemsHandler {
+	constructor() {
+		emitter.onAction('websocket.message', ({ client, message }) => {
+			if ((message?.type ?? '').toUpperCase() !== 'ITEMS') return;
+			try {
+				const parsedMessage = WebSocketItemsMessage.parse(message);
```

Comment:
When I send `{ "type": "items", "action": "delete" }` this line throws en error which is not handled within the catch block. It's an error from Zod which basically says `"path":["collection"],"message":"Required"`.

If I add the `collection` attribute, the error is handled correct though